crates/ruff/src/rules/airflow/rules/task_variable_name.rs

```diff
@@ -0,0 +1,102 @@
+use rustpython_parser::ast;
+use rustpython_parser::ast::{Expr, Ranged};
+
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::prelude::Constant;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks that the task variable name matches the `task_id` value for
+/// Airflow Operators.
+///
+/// ## Why is this bad?
+/// When initializing an Airflow Operator, for consistency, the variable
+/// name should match the `task_id` value. This makes it easier to
+/// follow the flow of the DAG.
+///
+/// ## Example
+/// ```python
+/// from airflow.operators import PythonOperator
+///
+///
+/// incorrect_name = PythonOperator(task_id="my_task")
+/// ```
+///
+/// Use instead:
+/// ```python
+/// from airflow.operators import PythonOperator
+///
+///
+/// my_task = PythonOperator(task_id="my_task")
+/// ```
+#[violation]
+pub struct AirflowVariableNameTaskIdMismatch {
+    task_id: String,
+}
+
+impl Violation for AirflowVariableNameTaskIdMismatch {
```

Comment:
I renamed this to prefix with "Airflow" to match the "Django" and "NumPy" conventions.